components/eam/cime_config/config_compsets.xml

```diff
@@ -304,6 +310,7 @@
     <entry id="CALENDAR">
       <values>
 	<value  compset="GEOS_">GREGORIAN</value>
+   <value compset="EAM%*SCAM_">TRUE</value>
```

Comment:
Is this going to affect non-MMF SCM cases?
Also, it looks like you didn't line up the white space with the lines around it.

Reply:
good catch. I need to replace TRUE with GREGORIAN. Yes it will affect non-MMF SCM behavior as well because all SCM cases need to use gregorian calendar. I will align the white space.

Reply:
I changed the compset condition to 'EAM%MMF[12XO]*-SCM'. So it won't affect non-MMF behavior.